muikku-atests/src/test/java/fi/otavanopisto/muikku/ui/base/discussions/DiscussionTestsBase.java

```diff
@@ -163,7 +163,7 @@ public void discussionReplyAndSubscribeTest() throws Exception {
     try {
       Discussion discussion = createDiscussion(discussionGroup.getId(), "test discussion");
       DiscussionThread thread = createDiscussionThread(discussionGroup.getId(), discussion.getId(), "Testing",
-          "<p>Testing testing daa daa</p>", false, false);
+          "<p>Testing testing daa daa</p>", false, null);
```

Comment:
Test failure on line 170. Check with front-end/testing whether `icon-bookmark-full` has been removed from the front-end.

Reply:
Does this test failure happen in github actions also? Every UI test seems to complete witrh success currently.